search/search_indexes.py

```diff
@@ -137,6 +136,15 @@ def prepare_preview_url(self, obj):  # pylint: disable=no-self-use
             run=data["run"],
         )
 
+    def prepare_titlesort(self, obj):  # pylint: disable=no-self-use
+        """Define what goes into the "titlesort" index."""
+        title = obj.title.strip()
+        # hack to handle empty titles
+        # to show up at the bottom of the sorted list
+        if not title:
+            title = 'z' * 30
```

Comment:
Why not prepend a '0' to all titles and then make the title a '1' if it's empty? That guarantees it's at the bottom. We can't rely on ASCII comparisons like this, what if the title starts with a non alphanumeric letter, or something unicode?